modules/nested-elements/assets/js/editor/nested-repeater/models/nested-model-base.js

```diff
@@ -22,7 +22,10 @@ export default class NestedModelBase extends elementor.modules.elements.models.E
 
 		return 'container' === childElType &&
 			'widget' === parentElType &&
-			isWidgetSupportNesting( this.get( 'widgetType' ) );
+			isWidgetSupportNesting( this.get( 'widgetType' ) ) &&
+			// When creating a container for the tabs widget specifcly from the repeater, the container should be locked,
+			// so only containers that are locked (created from the repeater) can be inside the tabs widget.
```

Comment:
// When creating a container for the tabs widget specifically from the repeater, the container should be locked,
			// so only containers that are locked (created from the repeater) can be inside the tabs widget.
```